modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss

```diff
@@ -186,3 +182,20 @@ a {
     }
   }
 }
+
+// Menu styles
+[class^="su-local-footer__"],
+.su-secondary-nav {
+  a {
```

Comment:
@sherakama,
Thanks for your help on this. This is more specific, but kinda ugly. I'm wondering if you have any suggestions?

Reply:
Does `.su-local-footer` not work?

Reply:
or `.su-local-footer__columns`

Reply:
Yes, @sherakama, they will work. 
I was trying to find a way to only target the menus, which, clearly I'm not 😭 
If someone were to put another block into this space, I didn't want to target any links in anything but the menu blocks. Is there a rule like:

[class^="su-local-footer__" && class$="links"]
Since I want to target :
`su-local-footer__primary-links`
`su-local-footer__secondary-links`
... and anything else with `su-local-footer__*-links`

Reply:
What about nav[class$="-links"] ?

Reply:
Thank you @jenbreese. That helped me think through this!

Reply:
@jenbreese Ready for your review!

Reply:
@shea
In this ticket: https://stanfordits.atlassian.net/browse/D8CORE-3146
Ishi is changing the focus and hover state on the secondary nav from red to black
I see that these colors are set here:
https://github.com/SU-SWS/decanter/blob/master/core/src/scss/utilities/variables/components/_secondary-nav.scss#L97
Should these color change go back to Decanter? Or should they stay here?

@jenbreese FYI

Reply:
I will defer to @kerri-augenstein for visual changes such as color. I would like them to be in the Decanter main project if we can, but it will need @kerri-augenstein's blessing.

Reply:
I spoke with @kerri-augenstein and @yvonnetangsu regarding the color changes. They are on board with the changes we’ve noted.

Reply:
@jenbreese,
this is ready for review. 
Most of this PR targets the external link styles which cannot be moved to decanter because they rely on the external link module. 
Anything else that can be moved to Decanter will be done in a separate ticket and PR. 

See https://stanfordits.atlassian.net/browse/D8CORE-3170

Reply:
worked fine for me. for SOE and Core